model/command_response.go

```diff
@@ -32,31 +33,38 @@ type CommandResponse struct {
 }
 
 func CommandResponseFromHTTPBody(contentType string, body io.Reader) (*CommandResponse, error) {
-	if strings.TrimSpace(strings.Split(contentType, ";")[0]) == "application/json" {
-		return CommandResponseFromJSON(body)
-	}
-	if b, err := io.ReadAll(body); err == nil {
-		return CommandResponseFromPlainText(string(b)), nil
+	if bodyRaw, err := io.ReadAll(body); err != nil {
+		return nil, nil
+	} else if commandResponse, err := CommandResponseFromJSON(bodyRaw); err == nil {
+		return commandResponse, nil
+	} else if ContentTypeIs(contentType, "application/json") {
+		return nil, err
+	} else {
+		return &CommandResponse{
+			Text: string(bodyRaw),
+		}, nil
 	}
```

Comment:
Although the current code here is compact, it's a little hard to scan through with all the indentation. I'm thinking we can expand the function a bit to let the code breathe a bit more.

```suggestion
	bodyRaw, err := io.ReadAll(body)
	if err != nil {
		return nil, nil
	}
	
	commandResponse, err := CommandResponseFromJSON(bodyRaw)
	if err == nil {
		return commandResponse, nil
	}
	
	if ContentTypeIs(contentType, "application/json") {
		return nil, err
	}
	
	return &CommandResponse{
		Text: string(bodyRaw),
	}, nil
```

Reply:
fixed